Cargo.toml

```diff
@@ -20,6 +20,7 @@ url = { version = "2", features = ["serde"] }
 uuid = { version = "1", features = ["serde", "v7"] }
 tracing = "0.1"
 http = "1"
+rcgen = { version = "0.13", features = ["aws_lc_rs"] }
```

Comment:
we have `ring` active on AC i wonder how this could affect us

Reply:
?

Reply:
Good point, I think this will collide :( 

`rustls` uses now `aws_lc_rs` by default (which supports FIPS). I don't remember why we decided to use Ring. I'll dig into this, and we can discuss it once you are back ;-)